examples/c/ex_col_store.c

```diff
@@ -0,0 +1,582 @@
+/*-
```

Comment:
It is good to refer to this test file in the column store documentation. Create another ticket and do the doc changes.